### PR TITLE
add steps to add site shortcut for Firefox

### PR DESCRIPTION
**Pull request by Klrfl**
Hi, I added steps to add `a.stro.cc` as a site shortcut in Firefox. I also changed the headings for better readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,23 @@ Use this as a site shortcut in Chrome!
...
+
+Firefox out of the box does not support adding a new search engine. However, there is an option to enable it. Here is how:
+
+- Open a new tab and navigate to `about:config`
+- In the search box, type `browser.urlbar.update2.engineAliasRefresh`.
+- At this point, the result list should be empty.
+  - If it is empty, click the **+** button at the top right.
+  - If the option already exists, set it to `true` by clicking the swap button on the right.
+- Now navigate to `about:preferences#search`.
+- Notice that it now has an **Add** button. Click on it.
```
Comment (delucis) [inline README.md]: Suggesting one small change here to be a bit clearer based on running through these steps myself:

```suggestion
- Click on the **Add** button under the “Search Shortcuts” list.
```

**Review by delucis — COMMENTED** [review]
Thanks for the extra docs @Klrfl! Left one small suggestion.

**Comment by Klrfl** [comment]
Thanks for the review! I already integrated your suggestion.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,23 @@ Use this as a site shortcut in Chrome!
...
+- Open a new tab and navigate to `about:config`
+- In the search box, type `browser.urlbar.update2.engineAliasRefresh`.
+- At this point, the result list should be empty.
+  - If it is empty, click the **+** button at the top right.
+  - If the option already exists, set it to `true` by clicking the swap button on the right.
+- Now navigate to `about:preferences#search`.
+- Click on the **Add** button under the “Search Shortcuts” list.
+- In the popup, set the **engine URL** to `https://a.stro.cc/%s`.
+- Set the alias to `astro`, or anything you prefer!
+- press **Add Engine**.
```
Comment (natemoo-re) [inline README.md]:
```suggestion
- Press **Add Engine**.
```

**Review by natemoo-re — APPROVED** [review]
This is great, thanks so much!

**Review by ElianCodes — APPROVED** [review]
Awesome! Thank you for adding this!

**Pull request merged**